ansible/roles/cli/tasks/download_cli.yml

```diff
@@ -20,4 +20,4 @@
     mode=0755
     validate_certs=False
     force=True
-  when: "'environments/docker-machine' in inventory_dir"
+  when: "'environments/docker-machine' and 'environments/mac' in inventory_dir"
```

Comment:
I think environments/docker-machine' and 'environments/mac should be "or" instead of and.